src/reaper_osara.cpp

```diff
@@ -3512,8 +3522,10 @@ void cmdRemoveOrCopyAreaOfItems(Command* command) {
 				// Translators: used for  "Item: Cut selected area of items" and "Item:
 				// Remove selected area of items".  {} is replaced by the number of items
 				// affected.
-				outputMessage(format(
-					translate_plural("selected area of {} item removed", "selected area of {} items removed", count), count));
+				s << format(
+					translate_plural("selected area of {} item removed", "selected area of {} items removed", count), count);
+				maybeAddRippleMessage(s, command->gaccel.accel.cmd);
+				outputMessage(s.str());
```

Comment:
nit: There's an overload of outputMessage that takes an ostringstream directly, so we can simplify this.

```suggestion
				outputMessage(s);
```